misc/util.py

```diff
@@ -0,0 +1,87 @@
+from spacy.tokens import DocBin, Doc
+from spacy.vocab import Vocab
+from typing import Dict, List
+from .notification import (
+    send_project_update,
+    send_project_update_throttle,
+)
+
+from submodules.model.enums import AttributeState
+from submodules.model.business_objects import (
+    attribute,
+    tokenization,
+    user,
+    organization,
+)
+from submodules.s3 import controller as s3
+
+
+def get_attribute_names_string(attribute_names: List[str]) -> str:
+    attribute_names = [f'"{name}"' for name in attribute_names]
+    attribute_names = ",".join(attribute_names)
+    return "{" + attribute_names + "}"
+
+
+def put_data_in_minio_bucket(project_id: str, missing_columns: List[str]) -> None:
+    missing_columns_str = ",\n".join(
+        ["'" + k + "',r.data->'" + k + "'" for k in missing_columns]
+    )
+    org_id = organization.get_id_by_project_id(project_id)
+    data = tokenization.get_doc_bin_table_to_json(project_id, missing_columns_str)
+    s3.upload_tokenizer_data(org_id, project_id, data)
+
+
+def __put_data_in_minio_bucket(project_id: str, missing_columns: List[str]) -> None:
+    missing_columns_str = ",\n".join(
+        ["'" + k + "',r.data->'" + k + "'" for k in missing_columns]
+    )
+    org_id = organization.get_id_by_project_id(project_id)
+    data = tokenization.get_doc_bin_table_to_json(project_id, missing_columns_str)
+    s3.upload_tokenizer_data(org_id, project_id, data)
```

Comment:
these two look awfully similar :)

Reply:
lol, did not see it myself for some reason, removed duplication